hack/run-e2e-test

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -uo pipefail
+
+OS_ARCH=$(go env GOOS)-amd64
+TEST_ID=$RANDOM
+CLUSTER_NAME=test-cluster-$TEST_ID
+TEST_DIR=/tmp/efs-e2e-test
+BASE_DIR=$(dirname $0)
+REGION=${AWS_REGION-us-east-1}
+ZONES=${AWS_AVAILABILITY_ZONES-us-east-1a,us-east-1b,us-east-1c}
+K8S_VERSION=${K8S_VERSION-1.14.1}
+
+echo "Testing in region: $REGION and zones: $ZONES"
+
+KOPS_DOWNLOAD_URL=https://github.com/kubernetes/kops/releases/download/1.14.0-alpha.3/kops-$OS_ARCH
+KOPS_PATH=$TEST_DIR/kops
+KOPS_STATE_FILE=s3://k8s-kops-csi-e2e
+
+# Download kops if not yet
+if [[ ! -e $KOPS_PATH ]]; then
+  mkdir -p $TEST_DIR 
+  echo "Downloading KOPS from $KOPS_DOWNLOAD_URL to $KOPS_PATH"
+  curl -L -X GET $KOPS_DOWNLOAD_URL -o $KOPS_PATH
+  chmod +x $KOPS_PATH
+fi
+
+echo "Creating cluster $CLUSTER_NAME"
+CLUSTER_YAML_PATH=$TEST_DIR/$CLUSTER_NAME.yaml
+SSH_KEY_PATH=$TEST_DIR/id_rsa
+ssh-keygen -P csi-e2e -f $SSH_KEY_PATH
+
+$KOPS_PATH create cluster --state $KOPS_STATE_FILE \
+  --zones $ZONES \
+  --node-count=3 \
+  --kubernetes-version=$K8S_VERSION \
+  --ssh-public-key=$SSH_KEY_PATH.pub \
+  $CLUSTER_NAME.k8s.local
+  $KOPS_PATH get cluster --state $KOPS_STATE_FILE $CLUSTER_NAME.k8s.local -o yaml > $CLUSTER_YAML_PATH
+  $KOPS_PATH replace --state $KOPS_STATE_FILE -f $CLUSTER_YAML_PATH
+  $KOPS_PATH update cluster --state $KOPS_STATE_FILE $CLUSTER_NAME.k8s.local --yes
+
+# Wait for cluster creation
+while [[ 1 ]]; do
+  $KOPS_PATH validate cluster --state $KOPS_STATE_FILE
+  ret=$?
+  if [[ $ret -eq 0 ]]; then
+    break
+  else
+    echo "Waiting cluster to be created"
+    sleep 30
+  fi
+done;
+
+# Push test driver image
```

Comment:
Let's move this section till line 75 to be before kops create cluster. This is to avoid driver build failed after cluster creation. And turn on `set -e`. See https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/335/files#diff-db2615cb8e63f2d009711c2dfd5088e4R57 for a similar change

Reply:
done